### DISCUSSION TITLE
Double spaces in function parameters

### DISCUSSION THREAD

**Discussion by mreduar** [squizlabs/PHP_CodeSniffer#3680] [Q&A] [answered]
I would like phpcs to tell me or correct automatically if it is possible when there is a function with parameters with a single space between the variable type and the variable name. Here is an example to make it better understood.

```diff
      /**
      * Handle the incoming request.
      *
-     * @param \App\Http\Requests\User\UserUpdateRequest $request
+     * @param  \App\Http\Requests\User\UserUpdateRequest  $request
-     * @param \App\Models\User $user
+     * @param  \App\Models\User  $user
      * @return \Illuminate\Http\RedirectResponse
     */
    public function __invoke(UserUpdateRequest $request, User $user)
    {
        ...
```

Is it possible to do so? Can a third party package like [slevomat/coding-standard](https://github.com/slevomat/coding-standard) do it?

**Comment by mreduar** [comment]
Any suggestions? I would still like to know

**Comment by jrfnl** [comment, marked as answer]
Perfectly possible, but very non-standard, so I doubt there will be pre-existing sniffs doing this. You can always write your own.

**Reply by mreduar** [reply]
Thank you!
